src/views/admin/PersonDetailsView.vue

```diff
@@ -41,6 +42,40 @@
     personStore.errorCode = '';
     navigateToPersonTable();
   };
+  
+  const getZuordnungen: ComputedRef<Zuordnung[] | undefined> = computed(() => {
+    const zuordnungen: Zuordnung[] | undefined = personenKontextStore.personenuebersicht?.zuordnungen;
+
+    if (!zuordnungen) return;
+
+    const zuordnungenWithKlasse: Zuordnung[] = [];
+
+    for (const zuordnung of zuordnungen) {
+      if (zuordnung.typ === OrganisationsTyp.Klasse) {
+        const administrierendeZuordnung: Zuordnung | undefined = zuordnungen.find(
+          (z: Zuordnung) => z.sskId === zuordnung.administriertVon && z.typ !== OrganisationsTyp.Klasse,
+        );
+        if (administrierendeZuordnung) {
+          const klasseWithAdmin: Zuordnung = {
+            ...administrierendeZuordnung,
+            klasse: zuordnung.sskName,
+          };
+          zuordnungenWithKlasse.push(klasseWithAdmin);
+        }
+      } else {
+        zuordnungenWithKlasse.push(zuordnung);
+      }
+    }
+
+    const uniqueZuordnungen: Zuordnung[] = zuordnungenWithKlasse.filter(
+      (zuordnung: Zuordnung, index: number, self: Zuordnung[]) => {
+        const foundIndex: number = self.findIndex((z: Zuordnung) => z.sskId === zuordnung.sskId);
+        return foundIndex === index || 'klasse' in zuordnung;
+      },
+    );
```

Comment:
Could break in the following scenarios:
1. Person has different roles at the same organisation (e.g. teacher and admin)
2. Person has multiple Klassen at the same organisation

Maybe instead create a Map like `Map<SSKID, Zuordnung>` where 
```ts
type Zuordnung = {
    sskId: string;
    /// [...]
    klassen: string[];
    rollen: string[];
}
```
That way you can fill the map and create a better result